automated-comments/ruby/two-fer/splat_args.md

```diff
@@ -0,0 +1 @@
+Define an explicit `name` variable rather than using the splat argument keyword (`*`). This will make the code clearer to someone else reading it, and involves less manipulation to access the value of the argument.
```

Comment:
```suggestion
Define an explicit `name` variable rather than using the splat argument keyword (`*`). This makes the code clearer to read, and involves less manipulation to access the value of the argument.
```

It's not just for other people, but also for yourself in an arbitrary timeframe.